src/bin/nydus-image/main.rs

```diff
@@ -264,7 +264,7 @@ fn prepare_cmd_args(bti_string: String) -> ArgMatches<'static> {
                         .short("v")
                         .help("RAFS filesystem format version number:")
                         .required(true)
-                        .default_value("5")
+                        .default_value("6")
```

Comment:
Maybe also make nydusify to v6 by default.

Reply:
Plan to handle it in another PR:)